src/components/tiles/table/table-toolbar.tsx

```diff
@@ -3,23 +3,27 @@ import React from "react";
 import ReactDOM from "react-dom";
 
 import { DeleteSelectedButton, LinkTileButton, SetExpressionButton } from "./table-toolbar-buttons";
+import { MergeInButton } from "../../../plugins/data-card/components/data-card-toolbar-buttons";
```

Comment:
move this to soon-to-exist generic toolbar in components/shared and use it in both places (table and DataCard Deck from there)